conanfile.py

```diff
@@ -0,0 +1,60 @@
+import os
+from io import StringIO
+import re
+
+from conan import ConanFile
+from conan.errors import ConanException
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.build import can_run
+from conan.tools.files import copy, load
+
+class viamTfliteCpu(ConanFile):
+    name = "mlmodel-tflite"
+
+    license = "Apache-2.0"
+    url = "https://github.com/viam-modules/mlmodel-tflite"
+
+    package_type = "application"
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "shared": [True, False]
+    }
+    default_options = {
+        "shared": True
+    }
+
+    exports_sources = "CMakeLists.txt", "LICENSE", "src/*"
+
+    def set_version(self):
+        content = load(self, "CMakeLists.txt")
+        self.version = re.search("set\(CMAKE_PROJECT_VERSION (.+)\)", content).group(1).strip()
+
+    def configure(self):
+        # If we're building static then build the world as static, otherwise
+        # stuff will probably break.
+        # If you want your shared build to also build the world as shared, you
+        # can invoke conan with -o "&:shared=False" -o "*:shared=False",
```

Comment:
Not sure this comment is right. My reading of the code is that if we're building static we build the world as static, and if we're building shared, we build the world is shared, with no changes needed by the person invoking the compiler.